modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DeveloperKeysForDXP.testcase

```diff
@@ -121,4 +121,266 @@ definition {
 		}
 	}
 
+	@description = "Automation ID: LPS-166936 | Test Summary: Verify that the user with the Admin and Partner Manager roles can see the actions (Generate New - Deactivate - Actions Menu - Deactivate button)"
+	@ignore = "true"
+	@priority = "4"
+	test UserAdminAndPartnerManagerRoleCanSeeAndPerformAllActionsForDXP {
+
+		// Needs refactor since it depends on example Avangrid Portals account
+
+		task ("Go to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+		}
+
+		task ("Given a user signs in on OKTA to update the Customer Portal") {
+			CustomerPortal.loginOktaToUpdateCP();
+
+			Navigator.openURL();
+
+			ApplicationsMenu.gotoSite(key_site = "${testSiteName}");
+		}
+
+		task ("Change the user role in the project") {
+			Account.openAccountsAdmin();
+
+			CustomerPortal.accountTableProject(title = "NASA ICE");
+
+			Account.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+
+			Account.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Partner Manager",
+				accountUserName = "Test Test");
+		}
+
+		task ("Go to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+
+			Refresh();
```

Comment:
Is this refresh really required? Would you please confirm with @luanakarla about the process to add refresh?

Reply:
There is an open bug for this [LPS-168369](https://issues.liferay.com/browse/LPS-168369) . It will probably only be fixed in phase 2 of the project. But I made an improvement in these steps, after a request from Brian. It has just been merged, so @styller1 will be able to use it when he enters the master.